src/Pages/Uptime/Monitors/index.jsx

```
@@ -104,19 +110,38 @@ const UptimeMonitors = () => {
+
+	const activeFilter = [...filterLookup].find(([key]) => key !== undefined);
+	const field = activeFilter?.[1] || sort?.field;
+	const filter = activeFilter?.[0] || search;
+
+	const effectiveTypes = selectedTypes?.length ? selectedTypes : TYPES;
+
 	const [
 		monitorsWithChecks,
 		monitorsWithChecksCount,
 		monitorsWithChecksIsLoading,
 		monitorsWithChecksNetworkError,
```

ESLint

[error] 'monitorsWithChecksNetworkError' is assigned a value but never used.

(no-unused-vars)